Solutions/Delinea Secret Server/Data Connectors/DelineaSecretServer_CEF.json

```diff
@@ -1,42 +1,42 @@
 {
-    "id": "ThycoticSecretServer_CEF",
-    "title": "Thycotic Secret Server",
-    "publisher": "Thycotic, Inc",
-    "descriptionMarkdown": "Common Event Format (CEF) from Thycotic Secret Server ",
+    "id": "DelineaSecretServer_CEF",
+    "title": "Delinea Secret Server",
+    "publisher": "Delinea, Inc",
+    "descriptionMarkdown": "Common Event Format (CEF) from Delinea Secret Server ",
     "graphQueries": [
         {
             "metricName": "Total data received",
-            "legend": "ThycoticSecretServer",
-            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Thycotic Software\"\n| where DeviceProduct == \"Secret Server\"\n"
+            "legend": "CommonSecurityLog (Delinea Secret Server)",
```

Comment:
@gaurava-delinea, legend should be "DATATYPE_NAME", there should not be spaces in datatype name, please check and update, thanks.

Reply:
chnaged the legend from CommonSecurityLog (Delinea Secret Server) to CommonSecurityLog(DelineaSecretServer)